ansible/roles/grav/tasks/main.yml

```diff
@@ -0,0 +1,38 @@
+---
+- name: "Install dependencies"
+  apt: name={{ item }} update_cache=yes state=latest
+  with_items:
+    - 'unzip'
+    - 'openssl'
+    - 'zip'
+
+# Environment
+- name: create grav group
+  group:
+    name: "{{ grav_group }}"
+    state: present
+
+- name: create grav user
+  user:
+    name: "{{ grav_user }}"
+    group: "{{ grav_group }}"
+    uid: "{{ grav_uid }}"
+    createhome: no
+    shell: /sbin/nologin
+    comment: "{{ grav_user_comment }}"
+    state: present
+
+- name: Create grav install folder
+  file:
+    path: "{{ item }}"
+    owner: "{{ grav_user }}"
+    group: "{{ grav_group }}"
+    state: directory
+  with_items:
+  - "{{ grav_path }}"
+  - "{{ grav_tmp_path }}"
+  - "{{ grav_install_path }}"
+
+- include: setup-php.yml
+- include: setup-nginx.yml
+- include: setup-grav.yml
```

Comment:
Because the content of the sites needs to be authored and edited by people that may not want to go through all the hassle of using something like git.
We want to use a CMS so that humans can edit the content.

At some point I would like the OONI website to be refactored to use something like grav.